components/x-gift-article/src/ShareArticleDialog.scss

```diff
@@ -190,13 +200,32 @@
 	width: 100%;
 }
 
+.share-article-dialog__advanced-non-subscriber {
+	&--element {
+		margin-top: oSpacingByName('s4');
+	}
+}
+
 .share-article-dialog__advanced-sharing-options {
-	margin-left: oSpacingByName('s8') + 5;
-	margin-bottom: oSpacingByName('s4');
+	margin-block: oSpacingByName('s4');
+	&--element {
+		display: flex !important;
```

Comment:
Comment explaining why we need !important here? Without one, this will be difficult to rework in future!